src/osbuild-manifests/coreos.osbuild.aarch64.mpp.yaml

```diff
@@ -144,49 +144,50 @@ pipelines:
               references:
                 name:oci-archive:
                   name: coreos.ociarchive
-      - mpp-if: container_repo != ''
-        then:
-          type: org.osbuild.ostree.deploy.container
-          options:
-            osname:
-              mpp-format-string: '{osname}'
-            target_imgref:
-              mpp-format-string: '{container_imgref}'
-            mounts:
-              - /boot
-              - /boot/efi
-            kernel_opts:
-              - rw
-              - '$ignition_firstboot'
-          inputs:
-            images:
-              type: org.osbuild.containers
-              origin: org.osbuild.source
-              mpp-resolve-images:
-                images:
-                  - source: $container_repo
-                    tag: $container_tag
         else:
-          type: org.osbuild.ostree.deploy
-          options:
-            osname:
-              mpp-format-string: '{osname}'
-            remote: fedora
-            mounts:
-              - /boot
-              - /boot/efi
-            kernel_opts:
-              - rw
-              - '$ignition_firstboot'
-          inputs:
-            commits:
-              type: org.osbuild.ostree
-              origin: org.osbuild.source
-              mpp-resolve-ostree-commits:
-                commits:
-                  - ref: $ref
-                    remote:
-                      url: $repourl
+          mpp-if: container_repo != ''
```

Comment:
```sugestion
- mpp-if: container_repo != ''
```

Reply:
Works fine without it. :) I'm not sure if the `else` block supports taking an array, but anyway this matches all the other `else` blocks in this file.